justfile

```diff
@@ -62,6 +62,10 @@ install:
     just node-install
     just py-install
 
+# Install `ov`-based git hooks
+@install-hooks:
+    bash -c "cp ./docker/dev_env/hooks/* ./.git/hooks"
```

Comment:
We also have to install pre-push hooks here.

Reply:
Done in https://github.com/WordPress/openverse/pull/4343/commits/63237098756a2c6d934519e58f80a8a2a92a4eb2